CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v2.0.0...HEAD)
+# [Unreleased](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v3.0.0...HEAD)
+
+# [v3.0.0](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v2.0.0...v3.0.0)
+### BREAKING CHANGES
+* Renamed `ILoggerConfig` option `logTopic` to `appName` to stay more uniform with the addition of `secondaryApp[Name|Version]` fields (see **Added** below).
```

Comment:
I ended up making this a major version bump. Mostly because I wanted to change `logTopic` to `appName` to make it more uniform with all the `secondaryApp...` fields, but also the `appVersion` field. I am not set on the major bump and can change it if necessary. 

Also, [PCM-1736](https://inindca.atlassian.net/browse/PCM-1736) will land in this release.